stack/common.go

```diff
@@ -0,0 +1,31 @@
+package stack
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/url"
+)
+
+func ParseYAML(
+	file string,
+	parseFunc func([]byte, ...string) (interface{}, error),
+	parseFuncArgs ...string,
+) (interface{}, error) {
```

Comment:
Why does this return interface{}?

Reply:
I think the interface may not be necessary for what we're doing here, I might refactor it to something more simple

Reply:
It was for mutualise this function for parsing Stack & Language

Reply:
We will add this back in the next re-factoring PR